carball/json_parser/player.py

```diff
@@ -55,7 +55,10 @@ def create_from_actor_data(self, actor_data: dict, teams: List['Team']):
         else:
             actor_type = list(actor_data["Engine.PlayerReplicationInfo:UniqueId"]['unique_id']['remote_id'].keys())[0]
             self.online_id = actor_data["Engine.PlayerReplicationInfo:UniqueId"]['unique_id']['remote_id'][actor_type]
-        self.score = actor_data["TAGame.PRI_TA:MatchScore"]
+        try:
```

Comment:
Care to elaborate as to when a player does not have a score?

Reply:
When the json does not contain a score not sure what you want me to say here.
I did not write the json converter